client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsClient.cs

```diff
@@ -118,7 +118,7 @@ public async Task<WalletResponse> CreateWalletAsync(string blockchainType, strin
             }
             catch (ErrorResponseException ex) when (ex.StatusCode == HttpStatusCode.Conflict)
             {
-                return null;
+                throw new DuplicationWalletException();
```

Comment:
Would it be a good idea to add some detail about the accident into this exception?